package.json

```diff
@@ -60,7 +60,7 @@
     "@angular/platform-browser": "^7.2.13",
     "@angular/platform-browser-dynamic": "^7.2.13",
     "@angular/router": "^7.2.13",
-    "@hmcts/ccd-case-ui-toolkit": "2.67.0-performance-tabs",
+    "@hmcts/ccd-case-ui-toolkit": "3.0.1-Spinner-issue-in-demo-MO-unassigned-cases-page",
```

Comment:
I would have preferred the name to have a lowercase 's' on spinner, but this is a different PR